src/types/Id.ts

```diff
@@ -6,6 +6,7 @@ export type Id = number;
 export const idSchema: JSONSchemaType<Id> = {
 	type: 'integer',
 	minimum: 1,
+	maximum: 4294967295,
```

Comment:
You should add a test for this (somewhere, doesn't have to be on all endpoint IMHO)